client/home/lib/myteam/components/hometeamteammates/title.coffee

```diff
@@ -0,0 +1,91 @@
+kd    = require 'kd'
+React = require 'app/react'
+TeamFlux       = require 'app/flux/teams'
+ButtonWithMenu = require 'app/components/buttonwithmenu'
+capitalizeFirstLetter = require 'app/util/capitalizefirstletter'
+
+module.exports = class HomeTeamTeamMatesTitle extends React.Component
+
+  constructor: (props) ->
+
+    super props
+
+    @state = {
+      isMenuOpen: no
+      filterKey: 'all'
+    }
+
+    @filterValue = 'See All'
```

Comment:
No need to bind this into instance like this, instead move this to `state` and use it from `state` everywhere. That way you won't have to deal with modifying props or anything else, just write this to state, and use it from the state where you are passing this down to other Components.